src/Features/Core/Portable/InitializeParameter/AbstractAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -394,12 +394,10 @@ private TStatementSyntax CreateNullCheckStatement(DocumentOptionSet optionSet, S
             DocumentOptionSet optionSet, Compilation compilation, SyntaxGenerator generator,
             IParameterSymbol parameter, string methodName)
         {
-            var stringType = compilation.GetSpecialType(SpecialType.System_String);
-
             // generates: if (string.IsXXX(s)) throw new ArgumentException("message", nameof(s))
             var condition = (TExpressionSyntax)generator.InvocationExpression(
                                 generator.MemberAccessExpression(
-                                    generator.TypeExpression(stringType),
+                                    generator.TypeExpression(SpecialType.System_String),
```

Comment:
This overload doesn't respect user preference for `string` vs `String`. This was initially a bug that was fixed in https://github.com/dotnet/roslyn/pull/19242. Closing the PR